cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -79,6 +79,21 @@ spec:
         - /usr/local/bin/kube-proxy
         - --config=/var/lib/kube-proxy/{{ .ProxyConfigMapKey }}
         - --hostname-override=$(NODE_NAME)
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10256
```

Comment:
A little piece of advice, it maybe better to replace `10256` by `ports.ProxyHealthzPort`
/cc @SataQiu

Reply:
@SataQiu Thanks, will do.

Reply:
@SataQiu I see this value hardcoded all over the codebase in too many places. I agree with your point of view that is better to use constant but this tidy up should/it would better to be done at the same time across multiple pieces as part of a different task/PR. So I would prefer not to change it only here, if you don't have a hard objection.